### PR TITLE
PAS-5 Add order list

### PR DESCRIPTION
**Pull request by pawelkondraciuk**

### CODE REVIEW

**Comment by pawelkondraciuk** [comment]
[Rewrite OrderList to TailwindCSS](https://trello.com/c/UEuxVTE3/5-rewrite-orderlist-to-tailwindcss)

File: apps/frontend/src/app/OrderElement/OrderElement.tsx [hunk 1]
```diff
@@ -14,7 +13,11 @@ const StyledTextField = styled(TextField)`
 export default function OrderElement(props: { order: OrderModel }) {
   const order = props.order;
   const statusColor =
```
Comment (lukszymanski) [inline apps/frontend/src/app/OrderElement/OrderElement.tsx:15]: or maybe, just maybe
```
const STATUS_COLORS = {
 [OrderStatus.InProgress]: 'primary',
 [OrderStatus.Ordered]: 'warning',
 [OrderStatus.Success]: 'success',
};

const statusColor = STATUS_COLOR[order.status];
```

File: libs/pages/dashboard/src/lib/order-list/order-list.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import { getOrderStatus, OrderModel } from '@pasnik/api/data-transfer';
+import {
+  CashIcon,
+  LocationMarkerIcon,
+  CalendarIcon,
+} from '@heroicons/react/outline';
+import { formatDistance } from 'date-fns';
+import { pl } from 'date-fns/locale';
+
+/* eslint-disable-next-line */
```
Comment (lukszymanski) [inline libs/pages/dashboard/src/lib/order-list/order-list.tsx:11]: eslint disable is not needed anymore i guesss

File: libs/pages/dashboard/src/lib/pages-dashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+  const handleTabGroupChange = useCallback((index) => {
+    setCurrentTab(index);
+  }, []);
+
+  const createOrderHandler = useCallback(() => {
+    history.push('/create-order');
+  }, [history]);
+
+  return (
```
Comment (lukszymanski) [inline libs/pages/dashboard/src/lib/pages-dashboard.tsx:54]: side note, divide this component to smaller ones

File: apps/frontend/src/app/OrderElement/OrderElement.tsx (hunk 1 above)
Comment (pawelkondraciuk) [reply apps/frontend/src/app/OrderElement/OrderElement.tsx:15]: Ignore OrderElement (part of next task)

Comment (lukszymanski) [reply apps/frontend/src/app/OrderElement/OrderElement.tsx:15]: 👍

File: libs/pages/dashboard/src/lib/pages-dashboard.tsx (hunk 3 above)
Comment (pawelkondraciuk) [reply libs/pages/dashboard/src/lib/pages-dashboard.tsx:49]: Done

File: libs/pages/dashboard/src/lib/dashboard-tabs/dashboard-tabs.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+export function DashboardTabs({ tabs }: DashboardTabsProps) {
+  return (
+    <Tab.List>
+      {tabs.map((tab) => (
+        <Tab key={tab.name} as={Fragment}>
+          {({ selected }) => (
+            <button
+              key={tab.name}
+              className={classnames(
+                selected
```
Comment (lukszymanski) [inline libs/pages/dashboard/src/lib/dashboard-tabs/dashboard-tabs.tsx:18]: fyi; you could use classnames like
```
'border-purple-500 text-purple-600': selected,
'border-transparent text-gray-500 hover:text-gray-700 hover:border-gray-200': !selected,
```

**Review by lukszymanski — APPROVED** [review]

**Pull request merged**